### DISCUSSION TITLE
Projects with low Build Activity - Survey

### DISCUSSION THREAD

**Discussion by jadsonjs** [mastodon/mastodon#17185] [General]
Dear Developers of project **mastodon/mastodon**,

We are conducting a study on CI metrics and we noticed that in your project, something peculiar happened.

During the period you are using Travis CI, the project passed by a long period of inactivity.
We noticed that the TravisCI configuration file (.travis.yml) was committed to your project on the date of **2016-03-18**.
But there is only a build history in TravisCI for a very short period, between **2018-03-30** and **2018-04-20**, in which were performed suddenly **343** builds.

Please, we would like you to answer this short survey with just 4 questions to help us identify the cause of this behavior.

https://forms.gle/UkkFrxpi7MNscYmV7

**Comment by Gargron** [comment]
We use CircleCI.
